source/stable-release_iot/user-guide-a12.rst

```diff
@@ -0,0 +1,410 @@
+
+	# Extract files
+	$ unzip sriov_patches.zip
+	$ unzip -jo ubuntu_kvm_multios_scripts.zip
+	$ tar xzvf caas-releasefiles-userdebug.tar.gz
+	
+	# Make all the script file executable
+	$ chmod +x <workspace>/*.sh
+	
+	# This will install kernel and firmware, and update grub
+    # If prompted, answer y to go ahead with changes
+	$ sudo ./sriov_setup_kernel.sh
+	
+	# After rebooting, check that the kernel is the installed version.
+	$ uname -r
+		5.15.71-lts2021-iotg
+
+Setup the Host OS for SRIOV
+***************************
+* Perform the setup for Ubuntu OS. The script is unzipped into ‘/home/$USER/’ directory
+
+.. code-block:: bash
+	# If prompted, answer y to go ahead with changes
+	$ sudo ./sriov_setup_ubuntu.sh
+
+	# Check if Host OS is running in SR-IOV PF mode
+	$ dmesg | grep SR-IOV
+		i915 0000:00:02.0: Running in SR-IOV PF mode 
+	# Check Host OS GuC and HuC Firmware Version
+	$ dmesg | grep GuC
+		i915 0000:00:02.0: [drm] GuC firmware i915/tgl_guc_70.bin version 70.5.1
+		i915 0000:00:02.0: [drm] GuC submission enabled
+	$ dmesg | grep HuC
+		i915 0000:00:02.0: [drm] HuC firmware i915/tgl_huc.bin version 7.9.3 
+		i915 0000:00:02.0: [drm] HuC authenticated
+
+	# Disable suspend and hibernate service
+	$ sudo systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
+
+	# Reboot Ubuntu host
+	$ sudo reboot now
+
+.. note::
+	If need to run any reliability or benchmark test, please run the commands below to disable auto suspend and hibernate on Ubuntu host
+
+Android Guest VM Setup
+**********************
+Follow the development environment set up instructions in
+`<https://docs.01.org/celadon/getting-started/build-source.html#set-up-the-development-environment>`_ for |C| build host setup.
+
+Users of Celadon-in-VM (CIV) release must ensure that Celadon platform host OS hardening measures are in place to ensure that the host OS could be treated as part of the secure computing base. This is essential to ensuring CIV security could be trusted in CIV operations.
+
+For Celadon Host OS hardening recommendations see:
+https://github.com/projectceladon/celadon-documentation/blob/master/source/getting-started/host-os-hardening.rst
+
+Celadon Source Requirements:
+
+* CIV_0X.XX.XX.XX_A11.xml
+
+From release package link: 
+https://www.intel.com/content/www/us/en/secure/design/confidential/software-kits/kit-details.html?kitId=738824&s=Newest 
+
+Build Celadon from Source :
+.. code-block:: bash
+	# Create symbolic link for Python if not already exists in ‘/usr/bin’ directory
+	$ sudo ln -s /usr/bin/python3 /usr/bin/python
+	
+Steps to sync to this release:
+
+.. code-block:: bash
+
+	# Create symbolic link for Python if not already exists in ‘/usr/bin’ directory
+	$ sudo ln -s /usr/bin/python3 /usr/bin/python
+
+	# Init with the default manifest
+	$ repo init -u https://github.com/projectceladon/manifest.git
+
+	# Copy the CIV manifest and use it
+	$ cp <source path>/CIV_0X.XX.XX.XX_A11.xml .repo/manifests/
```

Comment:
Manifest name needs updating

Reply:
Once The validation Freeze , We will Request for Final Manifest and Update the Same for Final Document Release